source/javascripts/lib/wirsing/filters/search.coffee

```diff
@@ -0,0 +1,26 @@
+#= require ../module
+
+# Returns a configurable search function that can be used
+# to filter an array of objects by searching for a term
+# within the specified property of each object.
+
+wirsing.module 'filter.search', ->
+
+  (options) ->
+
+    search = (items) ->
+      if search.term.length < search.minChars
+        items # dont filter
+      else
+        term = new RegExp search.term, "i" # case insensitive
+        items.filter (item) -> item[search.in].search(term) > -1
```

Comment:
Probably a simpler approach would be to write `item[search.in].toLowerCase.search(term.toLowerCase)`. This would save you from all the regex sanitation stuff.